apps/ledger-live-mobile/src/screens/AnalyticsOptInPrompt/variantA/Main.tsx

```diff
@@ -34,81 +40,86 @@ function Main() {
   const { navigateToMoreOptions, clickOnRefuseAll, clickOnAcceptAll, clickOnLearnMore } =
     useAnalyticsOptInPrompt();
 
-  const trackable = [t("analyticsOptIn.variantA.main.content.able.diagAndUsage")];
+  const trackable = [
+    t("analyticsOptIn.variantA.main.content.able.diagAndUsage"),
+    t("analyticsOptIn.variantA.main.content.able.personnalizationData"),
+  ];
 
   const unTrackable = [
     t("analyticsOptIn.variantA.main.content.unable.adresses"),
     t("analyticsOptIn.variantA.main.content.unable.balance"),
-    t("analyticsOptIn.variantA.main.content.unable.nfts"),
     t("analyticsOptIn.variantA.main.content.unable.personnalInfos"),
   ];
 
   return (
-    <Container alignItems="center">
-      <View>
-        <Titles>
-          <Text variant="h3Inter" fontSize={24} fontWeight="semiBold" color="neutral.c100">
-            {t("analyticsOptIn.variantA.main.title")}
-          </Text>
-          <Text pt={2} fontSize={14} color="neutral.c80">
-            {t("analyticsOptIn.variantA.main.subtitle")}
-          </Text>
-        </Titles>
-        <Content>
-          <Flex>
-            <Text pt={2} color="neutral.c70">
-              {t("analyticsOptIn.variantA.main.content.able.title")}
+    <ScrollView contentContainerStyle={{ flexGrow: 1 }} showsVerticalScrollIndicator={false}>
```

Comment:
As you do this multiple times (in different files)  you can create a reusable container ?